packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -141,23 +126,34 @@ export function PanelChrome({
   );
 
   return (
-    <div
-      className={cx(styles.container, { [styles.regularHeader]: hasHeader })}
-      style={containerStyles}
-      aria-label={ariaLabel}
-    >
+    <div className={styles.container} style={containerStyles} aria-label={ariaLabel}>
       <div className={styles.loadingBarContainer}>
         {loadingState === LoadingState.Loading ? <LoadingBar width={width} ariaLabel="Panel loading bar" /> : null}
       </div>
 
-      {(hoverHeader || !hasHeader) && menu && (
-        <HoverWidget menu={menu} title={title} offset={hoverHeaderOffset} dragClass={dragClass}>
-          {headerContent}
-        </HoverWidget>
+      {hoverHeader && (
+        <>
+          {menu && (
+            <HoverWidget menu={menu} title={title} offset={hoverHeaderOffset} dragClass={dragClass}>
```

Comment:
I think the hover header should be able to be rendered with or without a menu. E.g. for users who only want to have a drag and drop on their panel with no header. 
But this is probably for a different PR. cc @axelavargas

Reply:
Yea, agree but no change for that in this PR so something we can fix later if we have a need